### DISCUSSION TITLE
Update 2 timestampz on Update

### DISCUSSION THREAD

**Discussion by JSmithOner** [supabase/supabase#7911] [Questions] [answered]
When the table "config" is updated I need to update the two columns "update_date" and "configs_update_date" at the same time. Only problem this makes a recursive call.

Heres my function which is triggered on `UPDATE`:
```

BEGIN
    UPDATE editor_configs
    SET NEW.update_date = now(),
    NEW.configs_update_date = now()
    WHERE configs.project_uid in (old.project_uid, new.project_uid);
    RETURN NEW;
END;
```

How can I avoid this, I saw the `pg_trigger_depth() < 1` functionality but it appears I need to put it in the trigger definition and not in the function but I don't know where.

Many thanks in advance.

**Comment by wcooperscott** [comment, marked as answer]
This[ StackOverflow post](https://dba.stackexchange.com/a/163152) demonstrates how you can set `pg_trigger_depth() < 1`.

Looks like this is the example code you are looking for:
```
CREATE TRIGGER my_trigger
AFTER INSERT ON my_tbl
FOR EACH ROW
WHEN (pg_trigger_depth() < 1)
EXECUTE PROCEDURE my_trigger_func();
```
Run the code in the SQL Editor. Of course, you need to change the variables `my_trigger`, `my_tbl`, `my_trigger_func` to your project.

**Reply by JSmithOner** [reply]
Thanks so I need to put this in the SQL Editor goit, DO I have record of the queries I've made?

**Reply by wcooperscott** [reply]
@JSmithOner From the website, when looking at your project instance, you can select `Database > Postgres Logs` to view your Postgres activity.

**Reply by JSmithOner** [reply]
@wcooperscott Thanks
